index.html

```diff
@@ -209,54 +209,40 @@ <h2 class="gray-underline">How We Work</h2>
 <div class="section">
     <div class="container">
         <div class="title-header">
-            <h2>Our values</h2>
+          <h2>Software Craftsmanship</h2>
+          <p>Software Craftsmanship is at the heart of everything we do. We constantly hone our skills by teaching and learning from others within our company and the wider software development community.</p>
         </div>
-        <div class="row content-boxes">
-            <div class="col-md-4 md-margin-bottom-30">
-                <i class="fa fa-trophy fa-3x"></i>
-                <div class="content-box">
-                    <h3 class="heading-sm">QUALITY</h3>
-                    <p>We don't negotiate on quality. Software must deliver more value with time and not be a source of increasing pain and cost.</p>
+            <div class="row news-v1">
+                <div class="col-md-4 md-margin-bottom-40">
+                    <div class="news-v1-in">
+                        <img class="img-responsive" src="/assets/img/custom/home/the-software-craftsman-book.jpg" alt="">
+                        <h3><a href="#">The Software Craftsman: by our Co-founder Sandro Mancuso</a></h3>
+                        <p>The book is described as "an encyclopedia on the behaviours, attributes, and structure of an organisation that adheres to principles of Software Craftsmanship". A must read for anyone wanting to better understand the values behind the movement.</p>
+                        <ul class="list-inline news-v1-info no-margin-bottom">
+                            <li><a href="https://www.goodreads.com/book/show/23215733-the-software-craftsman">Discover more on Good Reads ...</a></li>
```

Comment:
Generally - opening links in a new tab is not good user experience. Although I don't feel too strongly about this in this case because it's a separate website. Lets stick with them as is for now and change later if others have a stronger opinion.